Docs/Practitioners/README.md

```diff
@@ -0,0 +1,13 @@
+# Practitioners Resource
+
+A practitioner is a medical professional. 
+The MED API allows for gathering information about a practitioner. 
```

Comment:
This should be consolidated into a single, flowing paragraph or simplified to a single sentence.